Automattic/wp-calypso
# Apps: Add Full Site Editing Plugin and Theme

#### Changes proposed in this Pull Request

* Add a new `full-site-editing` app containing `full-site-editing-plugin` and `blank-theme`.
* For demonstration purposes, add the Page Content preview block to the plugin.

#### Testing instructions

* From the Calypso root, try building the plugin with `npx lerna run build --scope='@automattic/full-site-editing'`.
* Move the `full-site-editing-plugin` folder (or symlink it) into `/wp-content/plugins` of any WordPress install.
* Open its dashboard, and enable the `Full Site Editing` plugin.
* Open the block editor, and insert the "Page Content Preview" block.
* Pick a page (you need to have pages first!), and check that it gets previewed inside the block.

Note: on WPCOM we need a file in the plugins root to load it.
It's enough to just use it to require the actual plugin php file, e.g.
```php
require_once( 'full-site-editing-plugin/full-site-editing-plugin.php' );
```
